### ISSUE TITLE
What does "real_image.requires_grad = True" mean?

### ISSUE THREAD

**Issue by MinhHuuNguyen** [rosinality/style-based-gan-pytorch#84]
Hi rosinality,
I don't know what "real_image.requires_grad = True" in the train.py means and why we need to backprop?
Thank you!

**Comment by rosinality** [comment]
It is for R1 gradient penalty. You can refer to https://arxiv.org/abs/1801.04406.
